src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs

```diff
@@ -38,7 +38,8 @@ public static partial class PlatformDetection
         public static bool IsNetBSD => RuntimeInformation.IsOSPlatform(OSPlatform.Create("NETBSD"));
         public static bool IsAndroid => RuntimeInformation.IsOSPlatform(OSPlatform.Create("ANDROID"));
         public static bool IsNotAndroid => !IsAndroid;
-        public static bool IsNotAndroidX86 => !(IsAndroid && IsX86Process);
+        public static bool IsAndroidX86 => IsAndroid && IsX86Process;
+        public static bool IsNotAndroidX86 => !IsAndroidX86;
```

Comment:
we don't seem to be using this

Reply:
It's used in a few places to disable some other tests (for example [src/libraries/System.Runtime/tests/System/DoubleTests.cs:1489](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Runtime/tests/System/DoubleTests.cs#L1489))

Reply:
ah, I missed it was already there :)